borgmatic/config/schema.yaml

```diff
@@ -1602,6 +1602,88 @@ properties:
                 example:
                     - start
                     - finish
+    zabbix:
+        type: object
+        additionalProperties: false
+        properties:
+            itemid:
```

Comment:
Hi, that is correct. In the zabbix API it is referred to as itemid. Which should I use?

Reply:
My take on this is: If the Zabbix `itemid` is a user- or developer-facing term (e.g., not just in the API), then it's fine as-is. But if it's an API implementation detail that a Zabbix user wouldn't necessarily be familiar with, then I'd use the more canonical `item_id`. Either way, it's not a huge deal.

Reply:
I think most will not used itemid anyways as it is not visible in the gui. I added it in since it technically is supported via the API.

Zabbix does reference this as `itemid` in their docs if that helps. [Link](https://www.zabbix.com/documentation/7.0/en/manual/api/reference/item/object#item)

Reply:
Okay, follow your heart then. :smile: